packages/mgt-react/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@microsoft/mgt-react",
+  "version": "1.3.0",
```

Comment:
Should the react package version match the consumed mgt version?

Reply:
I chose to match the minor version, but not the patch version initially. My reasoning is that if they are the same version, a bug fix in mgt will force a release of mgt-react, even though nothing changed. A patch release of mgt should not introduce new features, so it should not affect mgt-react.

Alternatively, another option would be to release mgt-react as same version as mgt, but not force an update to mgt-react whenever mgt has a patch release. Ex, releasing mgt-react today will give it version 1.3.4, but if mgt releases a 1.3.5, 1.3.6, 1.3.7, mgt-react will remain at 1.3.4 unless we need to publish an update. At that point, mgt-react will catch up to mgt and use the same version at the point of release.

Reply:
I like your initial thinking. Matching minor version makes a ton of sense.